Create image-sizes.md

brentmackinnon opened the pull request:
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

### Changes proposed in this Pull Request:

<!-- If necessary, indicate if this PR is part of a bigger feature. Add a label with the format `focus: name of the feature [team:name of the team]`. -->

<!-- Describe the changes made to this Pull Request and the reason for such changes. -->

migrates [image sizing doc](https://woo.com/wp-admin/post.php?post=8734950&action=edit) from woo.com

<!-- Begin testing instructions -->

### How to test the changes in this Pull Request:

<!-- Include detailed instructions on how these changes can be tested. Review and follow the guide for how to write high-quality testing instructions. -->

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. this is a standalone markdown file that shouldn't affect any other code, and should only be at risk of failing linting. 
2. make sure the doc looks alright before merging.
3.

<!-- End testing instructions -->

### Changelog entry

<!-- You can optionally choose to enter a changelog entry by checking the box and supplying data. -->

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

<!-- Choose only one -->

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

<!-- Choose only one -->

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [x] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message <!-- Add a changelog message here -->

#### Comment <!-- If the changes in this pull request don't warrant a changelog entry, you can alternatively supply a comment here. Note that comments are only accepted with a significance of "Patch" -->

</details>

brentmackinnon committed:
Create image-sizes.md

lsinger reviewed (changes requested):
I think this PR is a good opportunity to also move over the Wiki page about thumbnail regeneration over to the docs folder. The current version of the image sizing article links to the Wiki article, and as we want to move those articles over anyway, we shouldn't introduce links to it now that we will soon have to fix.

brentmackinnon commented:
great catch, leif! thank you.

brentmackinnon committed:
adding thumbnail-image-regeneration.md

migrating thumbnail doc from wiki (https://github.com/woocommerce/woocommerce/wiki/Thumbnail-Image-Regeneration-in-3.3) so that the internal link can be updated in our image sizing document.

brentmackinnon committed:
Update image-sizes.md

brentmackinnon committed:
Update thumbnail-image-regeneration.md

brentmackinnon committed:
Update thumbnail-image-regeneration.md

brentmackinnon commented:
@lsinger I think this one is ready for you to take another look.

lsinger commented:
> @lsinger I think this one is ready for you to take another look.

Thanks for the heads up, @brentmackinnon. In the future, ideally (also) use the "re-request review" button to ensure the PR comes up in my [list of PRs to review](https://github.com/pulls/review-requested): 

![Screenshot 2023-12-18 at 12 59 24](https://github.com/woocommerce/woocommerce/assets/23619/d1448c29-a9d5-4054-9cf5-e811f94e4b3d)

(No need to do that now with this PR.)

lsinger commented on docs/theme-development/image-sizes.md:
```diff
@@ -33,6 +33,7 @@ Store owners will still be able to control aspect ratio and cropping (see below)
 ## Customize image sizes in the customizer
 
 The customizer houses the options which control thumbnails in WooCommerce. ![Settings in the customizer](https://woocommerce.files.wordpress.com/2017/12/imagefeature.png?w=712) If the theme is declaring image sizes, the top section will be hidden and only the cropping option will be visible. Changing the cropping option, or widths, will update the preview on the right side to show how things will ...
+The customizer houses the options which control thumbnails in WooCommerce. ![Settings in the customizer](https://woocommerce.files.wordpress.com/2017/12/imagefeature.png?w=712) If the theme is declaring image sizes, the top section will be hidden and only the cropping option will be visible. Changing the cropping option, or widths, will update the preview on the right side to show how things will ...
```
I'm not sure if an absolute link to `/docs/...md` will work once the markdown has been converted and published on the site. Pinging @rrennick for his expertise here -- is this the way to link between documents and if now, which way is?

lsinger commented:
Closing and re-opening PR to see if that helps the Playwright E2E test checks get started: 

![Screenshot 2023-12-18 at 13 11 12](https://github.com/woocommerce/woocommerce/assets/23619/a4cb0efc-7511-4793-95e2-3104f24254e6)
